master-runner/src/main/scala/de/thm/ii/fbs/services/DockerService.scala

```diff
@@ -20,8 +20,10 @@ object DockerService {
     val builder = Seq.newBuilder[String]
     builder += DOCKER_CMD
     builder += DOCKER_RUN
-    builder += DOCKER_REMOVE
+    //builder += DOCKER_REMOVE
```

Comment:
You should not simply comment this out as it is needed for the bash checker. You can simply add an option here if the SQL checker does not need this option.

Reply:
I just commented it out for debugging purposes.